src/haddock/cns/toppar/haddock.top

```diff
@@ -263,6 +263,7 @@ Residue CHX
 Residue ACE
 Residue CTN
 Residue DUM
+Residue SHA
```

Comment:
Naive question. General shape? Or the specific shape of this PR?

Reply:
General shape dummy particle